pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -502,6 +502,7 @@ private static Map<String, List<KitRequestShipping>> getKitRequestsByKitId(@NonN
                     }
                 }
             } catch (SQLException ex) {
+                ex.printStackTrace();
```

Comment:
This typically is not necessary since the exception is logged with the stack trace.

Reply:
I believe I added for some debug and missed removing it. thank you, cleaned up